DevReadMe.md

```diff
@@ -100,17 +108,19 @@ ACA-Py ships with both inbound and outbound transport drivers for `http` and `ws
 
 Most configuration parameters are provided to the the agent at startup. Refer to the `Running` sections above for details on listing the available command line parameters.
```

Comment:
Not in the changes but an extra `the` here 😄

Reply:
Thanks - fixed.